code/game/gamemodes/cult/ritual.dm

```diff
@@ -370,7 +370,6 @@ var/global/list/rune_list = list() // HOLY FUCK WHY ARE WE LOOPING THROUGH THE W
 	else
 		M.LAssailant = user
 
-		return
```

Comment:
No its okay to bundle bugfixes with bugfixes, you don't bundle them with features for the explicit reason that it effectively looks like you are 'taking hostage' the bugfix. It's quite literally one of the problems we try to avoid.